databricks/koalas/indexes.py

```diff
@@ -711,6 +711,83 @@ def symmetric_difference(self, other, result_name=None, sort=None):
 
         return result
 
+    # TODO: return_indexer
+    def sort_values(self, ascending=True):
+        """
+        Return a sorted copy of the index.
+
+        .. note:: This method is not supported for pandas when index has NaN value.
+                  pandas raises unexpected TypeError, but we support treating NaN
+                  as the smallest value.
```

Comment:
at this point,

i'm not pretty sure how should we handling this difference from pandas

for example, pandas raises `TypeError` when index has NaN like below.

```python
>>> pidx = pd.Index([10, None, 5])
>>> pidx
Index([10, None, 5], dtype='object')

>>> pidx.sort_values()
Traceback (most recent call last):
...
TypeError: '<' not supported between instances of 'NoneType' and 'int'
```

whereas in koalas.

```python
>>> kidx = ks.Index([10, None, 5])
>>> kidx
Float64Index([10.0, nan, 5.0], dtype='float64')

>>> kidx.sort_values()
Float64Index([nan, 5.0, 10.0], dtype='float64')
```

Reply:
oh, and it seems they're using `None` as is for `Index`,

whereas we cast `None` to `np.nan`, 

is it intended??